lib/reek/smells/control_parameter.rb

```diff
@@ -52,12 +52,12 @@ class ControlParameter < SmellDetector
       # :reek:FeatureEnvy
       def inspect(ctx)
         ControlParameterCollector.new(ctx).control_parameters.map do |control_parameter|
-          name = control_parameter.name.to_s
+          argument = control_parameter.name.to_s
           smell_warning(
             context: ctx,
             lines: control_parameter.lines,
-            message: "is controlled by argument #{name}",
-            parameters: { name: name })
+            message: "is controlled by argument #{argument}",
```

Comment:
Actually this should rather be "parameter" and not "argument" but I'd like to fix that in a separate PR.